src/Kros.AspNetCore/Kros.AspNetCore.csproj

```
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
-    <Version>2.8.0</Version>
+    <Version>2.8.1</Version>
```

It's new feature. So the minor version should be enlarged
```suggestion
    <Version>2.9.0</Version>
```

---

ok